CHANGELOG.md

```diff
@@ -1,178 +1,31 @@
-- fix import (arnaudon - [240e663](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/240e6630d6956eb174301ab34e4f67f143abef09))
-- Update test_io.py to add section_id and segment_id. (Stéphanie Lisa Battini - [a7af007](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/a7af0075108e1f406e7ed59ee2c4de95605df45e))
-- fix sparse linalg (arnaudon - [83d9ba3](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/83d9ba34cc052d320969cf31c72a3b0fa3c9bbc0))
-- 2 files added (Battini Stéphanie Lisa - [d963f1c](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/d963f1c1881c6473b187105b820d9e389d9def7e))
-- fix example (arnaudon - [65dae36](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/65dae360824ade3fdaab645c5122426092fea3c3))
-- Apply 1 suggestion(s) to 1 file(s) (Alexis Arnaudon - [a5de33d](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/a5de33d6dd04fd52ae96103200d137b31242528a))
-- multiprocessing backend (arnaudon - [047eacd](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/047eacd3de049ffbdb91e87de89a020562536c8b))
-- minor (arnaudon - [9a05b56](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/9a05b56970ff69b4b63600e3d01f521e12773ea6))
-- Update test_io.py (Stéphanie Lisa Battini - [2a9af14](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/2a9af14fdde75c9042caf0d60be2b5bd21c63696))
-- add e-mail address (Stephanie Battini - [41a778b](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/41a778b44e7877110962e839eb9d9bd63bc81375))
-- test (Stephanie Battini - [ad20eb1](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/ad20eb1642b1ca7d36ca4bccf112d10353578b03))
-- stupid sparse matrix slicing fixed (Alexis Arnaudon - [95b4888](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/95b48880261ed370b4f48e5e052ba3c0a09f8177))
-- delete \\*.json (Stephanie Battini - [d9c37dc](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/d9c37dc56d8bda57bdf87d7f5357031ecddde74e))
-- Include nse tox-docker.yml (jean - [eea483c](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/eea483c548889f42dd860ba37c0dfb49cf9cb21b))
-- Merge "vasculatureAPI integration" (Stéphanie Lisa Battini - [1467d82](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/1467d82111b175c3e5e78287cc07e838864d441c))
-- Merge "forgotten files/updates" (Stéphanie Lisa Battini - [5eb0886](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/5eb088635f8ecc0d31097fafdaba058122b52343))
-- Merge "static solution" (Stéphanie Lisa Battini - [dc72ce5](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/dc72ce5d122f0d54cc380ea4e1b6a2449660d950))
-- tiny bug (alexis arnaudon - [7eb1c93](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/7eb1c93c0f7b6d5439949a4845b3e16c6b74114a))
-- test git review (Battini Stéphanie Lisa - [7fd9dd5](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/7fd9dd52dafbeddddf16b94bcc531c634fb7683a))
-- lint (arnaudon - [e0afa2a](https://bbpgitlab.epfl.ch/molsys/bloodflow/commit/e0afa2a99aad204afac03fdee9dcc0cda3e91b0a))
+### v0.1.5
+=======
```

Comment:
Since this is markdown, I would not underline the versions.

Reply:
Ok!